src/components/avatar/avatar.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import type { Meta, StoryObj } from "@storybook/react-vite";
+import { Avatar } from "./index";
+
+const meta = {
+  title: "Example/Avatar",
```

Comment:
nit: let's already get rid off Example

```suggestion
  title: "Avatar",
```